"Cannot find module 'resend' imported from node.modules/svelte-email-tailwind/preview.index.js."

"Cannot find module 'resend' imported from node.modules/svelte-email-tailwind/preview.index.js."

Getting this error when attempting to exactly replicate the nodemailer config route portion. Do you need resend or do you have any advise how to resolve?

--- a/package.json
+++ b/package.json
@@ -47,7 +47,6 @@
     "postcss": "^8.4.31",
     "prettier": "^2.8.8",
     "prettier-plugin-svelte": "^2.10.1",
-    "resend": "^2.0.0",
     "svelte-check": "^3.6.0",
     "svelte-persisted-store": "^0.7.0",
     "tslib": "^2.6.2",
@@ -59,6 +58,7 @@
   "dependencies": {
     "@steveninety/html-to-text": "^9.0.5",
     "pretty": "^2.0.0",
+    "resend": "2.0.0",
     "svelte": "^3.59.2",
     "tw-to-css": "0.0.12"
   }

